Packs/AWSCompromisedEC2Response/Playbooks/playbook-AWS_Compromised_EC2_Response.yml

```diff
@@ -0,0 +1,618 @@
+      roleSessionName: {}
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: AWS - EC2
+      description: Creates an entry (a rule) in a network ACL with the specified rule
+        number.
+      id: 4e646bda-4203-4c68-8118-9195f1e66be0
+      iscommand: true
+      name: Create Block ACL Entry for Offending IP (Outbound)
+      script: AWS - EC2|||aws-ec2-create-network-acl-entry
+      type: regular
+      version: -1
+    taskid: 4e646bda-4203-4c68-8118-9195f1e66be0
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 300,
+          "y": 720
+        }
+      }
+  "16":
+    id: "16"
+    ignoreworker: false
+    note: false
+    quietmode: 0
+    scriptarguments:
+      CidrBlock:
+        complex:
+          accessor: ipAddressV4
+          root: incident.service.action.networkConnectionAction.remoteIpDetails
+          transformers:
+          - args:
+              prefix: {}
+              suffix:
+                value:
+                  simple: /32
+            operator: concat
+      Code: {}
+      DryRun: {}
+      Egress:
+        simple: "False"
+      From:
+        simple: "0"
+      Ipv6CidrBlock: {}
+      NetworkAclId:
+        simple: acl-0780ee09f0dbd6f7c
+      Protocol:
+        simple: "-1"
+      RuleAction:
+        simple: deny
+      RuleNumber:
+        simple: "1"
+      To:
+        simple: "65535"
+      Type: {}
+      region: {}
+      roleArn: {}
+      roleSessionDuration: {}
+      roleSessionName: {}
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: AWS - EC2
+      description: Creates an entry (a rule) in a network ACL with the specified rule
+        number.
+      id: f8d05e1d-1365-46ac-8255-1a90db970bbb
+      iscommand: true
+      name: Create Block ACL Entry for Offending IP (Inbound)
+      script: AWS - EC2|||aws-ec2-create-network-acl-entry
+      type: regular
+      version: -1
+    taskid: f8d05e1d-1365-46ac-8255-1a90db970bbb
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 300,
+          "y": 895
+        }
+      }
+  "17":
+    conditions:
+    - condition:
+      - - left:
+            iscontext: true
+            value:
+              complex:
+                accessor: actionType
+                root: incident.service.action
+          operator: isEqualString
+          right:
+            value:
+              simple: NETWORK_CONNECTION
+      label: "yes"
+    id: "17"
+    ignoreworker: false
+    nexttasks:
+      "yes":
+      - "15"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      id: e88eb14f-adc6-4b7c-8a9d-8da57ff33a54
+      iscommand: false
+      name: Confirm Action is "Network Connection"
+      type: condition
+      version: -1
+    taskid: e88eb14f-adc6-4b7c-8a9d-8da57ff33a54
+    timertriggers: []
+    type: condition
```

Comment:
Please add an else case leading to a `manual task` / `"Done"` task based on the use case.